CHANGELOG.md

```diff
@@ -1,5 +1,181 @@
 # Changelog
 
+## v0.67.0
+
+### Breaking
+
+- Breaking: Removed initialization of React Native polyfills and global variables from React renderers. ([b14b34b232](https://github.com/facebook/react-native/commit/b14b34b232b4873137ea3f3c7a90f86b9013885a) by [@rubennorte](https://github.com/rubennorte))
+
+#### Android specific
+
+#### iOS specific
+
+### Added
+
+- Make react-native depend on react-native-gradle-plugin ([80e24834ba](https://github.com/facebook/react-native/commit/80e24834ba8c581157fb6c9f5d8b49301677c487) by [@cortinico](https://github.com/cortinico))
```

Comment:
This change was reverted in a subsequent commit, so maybe should be placed lower in the list or removed.